dashboards/alert-policies.json

```diff
@@ -0,0 +1,1347 @@
+            "steps": [
+              {
+                "color": "green"
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          },
+          "unit": "none"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 6,
+        "w": 24,
+        "x": 0,
+        "y": 33
+      },
+      "id": 28,
+      "options": {
+        "colorMode": "value",
+        "graphMode": "area",
+        "justifyMode": "auto",
+        "orientation": "auto",
+        "percentChangeColorMode": "standard",
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "showPercentChange": false,
+        "textMode": "auto",
+        "wideLayout": true
+      },
+      "pluginVersion": "12.0.0",
+      "targets": [
+        {
+          "disableTextWrap": false,
+          "editorMode": "builder",
+          "expr": "gather_alert_policy_operation_failure{tenantId=\"$tenantId\", errorType=\"$errorType\"}",
+          "fullMetaSearch": false,
+          "includeNullMetadata": true,
+          "legendFormat": "__auto",
+          "range": true,
+          "refId": "A",
+          "useBackend": false
+        }
+      ],
+      "title": "Error per type: $errorType",
+      "type": "stat"
+    }
+  ],
+  "preload": false,
+  "refresh": "30s",
+  "schemaVersion": 41,
+  "tags": [],
+  "templating": {
+    "list": [
+      {
+        "current": {
+          "text": "1135adb2-3a41-4e96-b6c8-c12b106b13cc",
+          "value": "1135adb2-3a41-4e96-b6c8-c12b106b13cc"
+        },
+        "definition": "label_values(gather_alert_policies_counter_by_tenant,tenantId)",
+        "label": "Tenant ID",
+        "name": "tenantId",
+        "options": [],
+        "query": {
+          "qryType": 1,
+          "query": "label_values(gather_alert_policies_counter_by_tenant,tenantId)",
+          "refId": "PrometheusVariableQueryEditor-VariableQuery"
+        },
+        "refresh": 1,
+        "regex": "",
+        "type": "query"
+      },
+      {
+        "current": {
+          "text": "capture",
+          "value": "capture"
+        },
+        "definition": "label_values(gather_enabled_alert_policies_by_tenant,policyType)",
+        "hide": 2,
+        "includeAll": true,
+        "label": "Policy Type",
+        "name": "policyType",
+        "options": [],
+        "query": {
+          "qryType": 1,
+          "query": "label_values(gather_enabled_alert_policies_by_tenant,policyType)",
+          "refId": "PrometheusVariableQueryEditor-VariableQuery"
+        },
+        "refresh": 1,
+        "regex": "",
+        "type": "query"
+      },
+      {
+        "allValue": "All",
+        "current": {
+          "text": "conflict",
+          "value": "conflict"
+        },
+        "definition": "label_values(gather_alert_policy_operation_failure,errorType)",
```

Comment:
Oh, I think I missed this dropdown for selecting errorType values. Neat. You can probably ignore my previous comment about it then.